pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -668,6 +668,16 @@ type PodSandboxConfig struct {
 	//      * localhost/<profile-name>: the profile installed to the node's
 	//        local seccomp profile root
 	//
+	// 3. Sysctls
```

Comment:
Regardless of the decision in https://github.com/kubernetes/kubernetes/pull/36446, having these enumerated here will be helpful, so :+1: